plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -285,38 +350,60 @@ function WordInfoDialog:init()
         buttons = {{reset_button, remove_button}},
         show_parent = self
     }
+
+    local copy_button = Button:new{
+        text = "", -- copy in nerdfont,
+        callback = function() Device.input.setClipboardText(self.title) end,
```

Comment:
When tap on this "copy" icon, there is not feedback that something is happening - so I ended up wondering if it was just cosmectics. Dunno if we show a Notification "Stuff copied to clipboard" in other contexts (I think I've seen that, don't remember where... pinging @hius07 if we have some consistent behaviour with copy to clipboard that should be followed here.)

Reply:
Added a `UIManager:show(Notification:new{
                text = _("Word copied to clipboard."),
            })`, and it's already been used in `textinput`.